### PR TITLE
Add News entry "Mixxx 2.3.6 released"

### PR DESCRIPTION
**Pull request by daschuer**
Here is a first draft:

### CODE REVIEW

**Review by JoergAtGithub — APPROVED** [review]

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+title: Mixxx 2.3.6 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.6, release announcement
+status: draft
+comments: yes
+
+Welcome the Mixxx release version 2.3.6, the final maintenance release before releasing the upcoming 2.4.0 with exiting new features.
+
+This little bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user and ensuring a reliable mixing experience.
```
Comment (ronso0) [inline content/news/XXXX-XX-XX-mixxx-2-3-6-released.md]:
```suggestion
This small bug fix release may appear insignificant, but it manifests our effort to deliver rock-solid quality and ensure a reliable mixing experience.
```
'little' sounds a bit like 'cute'

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+This release improves among other things some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.
```
Comment (ronso0) [inline content/news/XXXX-XX-XX-mixxx-2-3-6-released.md]:
```suggestion
This release improves, among other things, some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.
```

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+You can download the new version here: https://mixxx.org/download/ There you will also find the 2.4 beta with which you can already try the new features that will be released soon.
```
Comment (ronso0) [inline content/news/XXXX-XX-XX-mixxx-2-3-6-released.md]:
```suggestion
You can download the new version from https://mixxx.org/download/ There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
```
nitpick: let's keep "2.4 beta" together. &nbsp; is a hack, ^+space should be preferred AFAIU

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Here is the full change log of Mixxx 2.3.6 for more details:
```
Comment (ronso0) [inline content/news/XXXX-XX-XX-mixxx-2-3-6-released.md]:
```suggestion
Here is the full changelog of Mixxx 2.3.6 for more details:
```

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  [#11760](https://github.com/mixxxdj/mixxx/pull/11760)
+  [#11545](https://github.com/mixxxdj/mixxx/issues/11545)
+* Don't reject key values from file metadata with non-minor/-major scales.
+  [#11001](https://github.com/mixxxdj/mixxx/pull/11001)
+  [#10995](https://github.com/mixxxdj/mixxx/issues/10995)
+* Allow playing tracks with durations of more than 6 h
+  [#11511](https://github.com/mixxxdj/mixxx/pull/11511)
+  [#10995](https://github.com/mixxxdj/mixxx/issues/10995)
+* Soundtouch latency compensation updated for version 2.1.1 to 2.3
+  [#11154](https://github.com/mixxxdj/mixxx/pull/11154)
```
Comment (ronso0) [inline content/news/XXXX-XX-XX-mixxx-2-3-6-released.md:75]: + the updated changelog from #11811 of course

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by ronso0** [comment]
Otherwise LGTM, thank you for starting this!

**Comment by ronso0** [comment]
Draft until mixxxdj/mixxx#11811 is merged

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md (hunk 1 above)
Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-mixxx-2-3-6-released.md]: I agree with the `s/little/small` and the `s/ensuring/to ensure/` but I prefer the "testament" version of the rest of the sentence. Alternatively
```suggestion
This small bug fix release may appear insignificant, but it serves as evidence of the rock-solid quality we bring to every user and to ensure a reliable mixing experience.
```

**Comment by daschuer** [comment]
Done.

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
+title: Mixxx 2.3.6 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.6, release announcement
+status: draft
+comments: yes
+
+Welcome the Mixxx release version 2.3.6, the final maintenance release before releasing the upcoming 2.4.0 with exiting new features.
+
+This small bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user and tp ensure a reliable mixing experience.
```
Comment (Swiftb0y) [inline content/news/XXXX-XX-XX-mixxx-2-3-6-released.md]:
```suggestion
This small bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user to ensure a reliable mixing experience.
```

**Review by Swiftb0y — CHANGES_REQUESTED** [review]
LGTM otherwise

**Review by Swiftb0y — APPROVED** [review]
LGTM. waiting for @ronso0 to give their blessings.

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+status: draft
+comments: yes
+
+Welcome the Mixxx release version 2.3.6, the final maintenance release before releasing the upcoming 2.4.0 with exiting new features.
+
+This small bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user to ensure a reliable mixing experience.
+
+This release improves, among other things, some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.
+
+You can download the new version from https://mixxx.org/download/ There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
```
Comment (ronso0) [inline content/news/XXXX-XX-XX-mixxx-2-3-6-released.md]: this is not turned into a hyperlink, at least not in the preview
https://deploy-preview-303--mixxx-website.netlify.app/news/2023-08-12-mixxx-2-3-6-released/

**Review by ronso0 — CHANGES_REQUESTED** [review]
LGTM otherwise, thank you!

File: content/news/XXXX-XX-XX-mixxx-2-3-6-released.md [hunk 8]
```diff
@@ -10,7 +10,7 @@ This small bug fix release may appear insignificant, but it's a testament to the
 
 This release improves, among other things, some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.
 
-You can download the new version from https://mixxx.org/download/ There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
+You can download the new version from the [Donload](https://mixxx.org/download/) page. There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
```
Comment (ronso0) [inline content/news/XXXX-XX-XX-mixxx-2-3-6-released.md]:
```suggestion
You can download the new version from the [Download](https://mixxx.org/download/) page. There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
```

**Comment by ronso0** [comment]
I took the liberty to fix the typo myself.

LGTM, thank you!

**Review by ronso0 — APPROVED** [review]

**Comment by ronso0** [comment]
Waiting for the smoke test results now

**Comment by daschuer** [comment]
Now we have a release and I will go ahead an merge this. Thank you for all contributions.

**Pull request merged**